src/main/java/uk/gov/hmcts/ccd/domain/model/definition/WorkbasketInputDefinition.java

```diff
@@ -37,4 +38,16 @@ public void setCaseTypeId(String caseTypeId) {
     public void setFields(List<WorkbasketInputField> fields) {
         this.fields = fields;
     }
+
+    /**
+     **/
+    @ApiModelProperty(value = "")
```

Comment:
Do we need the @ApiModelProperty(value = "") as value="" ?

Reply:
Purely added for consistency with rest of this class.